index.css

```diff
@@ -1,619 +1,511 @@
+    font-size: 72px;
+    font-weight: 400;
+    color: #ff6b00;
+  }
+
+  .details {
+    width: 327px;
+    height: 148px;
+    position: relative;
+    top: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: regular;
+    font-size: 16px;
+    color: #f4f5f7;
+  }
+
+  .resume .res {
+    display: flex;
+    padding: 10px;
+    border: 1px solid #ff6b00;
+    width: 150px;
+    color: #fff;
+    background-color: #ff6b00;
+    text-decoration: none;
+    font-size: 16px;
+    font-family: "Inter", sans-serif;
+  }
+
+  .about-me-img {
+    position: relative;
+    top: 160px;
+  }
+
+  .experience {
+    position: relative;
+    top: 230px;
+  }
+
+  .experience ul li {
+    width: 280px;
+    list-style: none;
+    text-align: center;
+    display: inline;
+  }
+
+  .experience ul li h4 {
+    font-size: 25px;
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-weight: 600;
+  }
+
+  .experience ul li img {
+    position: relative;
+    top: 35px;
+  }
+
+  .form-area {
+    display: flex;
+    flex-direction: column;
+    width: 375px;
+    height: 906px;
+    padding-top: 200px;
+    padding-left: 20px;
+    margin-left: 8px;
+    padding-right: -10px;
+    margin-right: 5px;
+  }
+
+  .form-area .creator h2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 90%;
+    height: 216px;
+    font-family: "Inter", sans-serif;
+    font-weight: 700;
+    font-size: 32px;
+    font-style: normal;
+    color: #172b4d;
+    line-height: 44px;
+    margin-bottom: 15%;
+  }
+
+  textarea {
+    width: 90%;
+    height: 114px;
+    margin-top: -20px;
+    border: none;
+    background-color: #fbf8f7;
+    padding-top: 10px;
+    font-family: "Inter", sans-serif;
+    font-style: italic;
+    font-size: 15px;
+    color: #3c3a39;
+  }
+
+  .field {
+    border-style: none;
+  }
+
+  .field input {
+    width: 90%;
+    height: 18px;
+    padding: 20px 0;
+    border-style: none;
+    border-bottom: 2px solid #e5e5e5;
+    margin-bottom: 40px;
+  }
+
+  form button {
+    width: 35%;
+    height: 48px;
+    align-content: center;
+    position: relative;
+    top: 25px;
+    left: 90px;
+    background-color: #ff6b00;
+    border: 2px solid #ff6b00;
+  }
+
+  form button:hover {
+    border-color: #f55800;
+    color: #f55800;
+    background-color: #fff5e1;
+  }
+
+  form button:disabled {
+    border-color: #f4f1f0;
+    color: #5b5957;
+    background-color: #f4f1f0;
+  }
+
+  form button:enabled {
+    background-color: #f55800;
+    color: #fff;
+  }
+
+  form button:active {
+    color: #f4f1f0;
+    background-color: #e05304;
+  }
+
+  form button h5 {
+    font-family: "Inter", sans-serif;
+    font-weight: 700px;
+    font-style: normal;
+    line-height: 24px;
+    font-size: 17px;
+    letter-spacing: 0.03em;
+    color: #fff;
+  }
+
+  .form-area hr {
+    width: 375px;
+    height: 1px;
+    position: relative;
+    top: 125px;
+    left: -28px;
+  }
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 use the color from the Figma design `#DFE1E6` to match with the original design.